### PR TITLE
docs: contribution guidelines and code of conduct

### LINKED ISSUE #50: docs: Contributing guidelines
- [x] Move the contribution guidelines out of the readme.
- [x] Clean up the readme.

### PR DESCRIPTION
**Pull request by cairin**
## Changes proposed in this pull request

- Adds:
  - Contribution guidelines
  - Code of conduct
- Cleans up readme

## Context

Closes #50 

## Checklist

- [x] Related issues linked using `fixes #number`
- [ ] Tests added/updated
- [x] Documentation added
- [ ] Make sure that all checks pass

### CODE REVIEW

File: .github/code_of_conduct.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+Community leaders have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, and will communicate reasons for moderation decisions when appropriate.
+
+## Scope
+
+This Code of Conduct applies within all community spaces, and also applies when an individual is officially representing the community in public spaces. Examples of representing our community include using an official e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported to the community leaders responsible for enforcement at opensource@github.com. All complaints will be reviewed and investigated promptly and fairly.
```
Comment (uchibeke) [inline .github/code_of_conduct.md:40]: Should we create an @interledger.org email for this? A @coil.com email will do too but @interledger.org might be a better option

Comment (cairin) [reply .github/code_of_conduct.md:40]: @traviscrist do we have an email for this sort of thing yet?

File: README.md [hunk 2]
```diff
@@ -38,166 +14,6 @@ Rafiki also allows for delegated access, offering OAuth-based flows to grant
 third-party applications access to Interledger functionality on a user's
 account.
```
Comment (uchibeke) [inline README.md:16]: A suggestion: Should we include links to the Interledger docs or something that introduces people to the different features of Interledger that Rafiki will expose? I like what they did here https://github.com/mozilla/glean.js

**Review by uchibeke — COMMENTED** [review]
LGTM ♠️ . I just had comments about the email we are using, we could use devs@coil.com to start, and adding info about Interledger for beginners

File: .github/code_of_conduct.md (hunk 1 above)
Comment (cairin) [reply .github/code_of_conduct.md:40]: Set it to devs@coil.com for now.

File: README.md (hunk 2 above)
Comment (cairin) [reply README.md:18]: I've added some general links to NB parts of the ecosystem. Maybe we should go over this part and flesh it out a bit more.

**Pull request merged**